config/src/main/java/com/typesafe/config/impl/SimpleConfig.java

```diff
@@ -482,15 +483,22 @@ public TemporalAmount getTemporal(String path){
 
     @Override
     public List<Long> getBytesList(String path) {
-        List<Long> l = new ArrayList<Long>();
+        return getMemorySizeList(path).stream()
+            .map(ConfigMemorySize::toBytes)
```

Comment:
here we'll also have changed from ConfigException.BadValue to IllegalArgumentException I think; a toBytes that takes the origin/path could solve.

Reply:
sorry, fixed, thank you.